libpod/runtime.go

```diff
@@ -739,6 +741,30 @@ func getLockManager(runtime *Runtime) (lock.Manager, error) {
 	return manager, nil
 }
 
+func probeConmon(conmonBinary string) error {
+	outdatedVersionError := errors.Errorf("Outdated conmon version. Please update to v1.0.0 or later")
```

Comment:
This should be a public constant, so API users can test for it

Reply:
Ideally, cut out the v1.0.0 part too - we can do an errors.Wrapf(ErrTooOldConmon, "please update to v1.0.0 or later")

Reply:
fixed, and added the reporting of that error